package/crossplane.yaml

```
@@ -1,7 +1,25 @@
 apiVersion: meta.pkg.crossplane.io/v1alpha1
 kind: Provider
 metadata:
-  name: equinix-metal
+  name: provider-equinix-metal
+  annotations:
+    company: Equinix Metal
+    maintainer: Equinix Metal Maintainers <support@equinixmetal.com>
+    source: github.com/packethost/crossplane-provider-equinix-metal
+    license: Apache-2.0
+
+    descriptionShort: |
+      The Equinix Metal Crossplane provider enables bare metal infrastructure
+      management for Equinix Metal.
+     
+    description: |
+      The Equinix Metal Crossplane provider adds support for managing Equinix
+      Metal resources in Kubernetes.
+    
+    readme: |
+      `provider-equinix-metal` is the Crossplane infrastructure provider for
+      [Equinix Metal](https://metal.equinix.com/).
+
```

@displague I think this is some good initial metadata to start with, we can update further if needed

---

```suggestion
  annotations:
    company: Equinix Metal
    maintainer: Equinix Metal Maintainers <support@equinixmetal.com>
    source: github.com/packethost/crossplane-provider-equinix-metal
    license: Apache-2.0

    descriptionShort: |
      The Equinix Metal Crossplane provider enables bare metal infrastructure
      management for Equinix Metal.
     
    description: |
      The Equinix Metal Crossplane provider adds support for managing Equinix
      Metal resources in Kubernetes.
    
    readme: |
      `provider-equinix-metal` is the Crossplane infrastructure provider for
      [Equinix Metal](https://metal.equinix.com/).

      Available resources and their fields can be found in the [CRD
      Docs](https://doc.crds.dev/github.com/packethost/crossplane-provider-packet).
```

Also would consider adding:
```
      If you encounter an issue please reach out on the #equinix-metal channel in
      [slack.crossplane.io](https://slack.crossplane.io) and create an issue in
      the [packethost/crossplane-provider-equinix-metal](https://github.com/packethost/crossplane-provider-equinix-metal)
      repo.
```

---

@prasek yeah I had temporarily left that out because the link is actually https://doc.crds.dev/github.com/packethost/crossplane-provider-packet .. Although it would hidden behind the link name so maybe that is fine?

---

think so for now, so people can browse the available types from the registry.

at some point would be cool for https://doc.crds.dev to automatically redirect renamed repos like GitHub does, but for now we could just add the new repo name to https://github.com/crdsdev/doc/blob/master/deploy/config.yaml and deprecate the old one until the Crossplane API docs link is cut over.

---

@prasek it will redirect to the correct repo, but the name will not automatically update. Very easy to fix manually though. It is just using the one provided to point to, then that is being redirected when we clone and parse

---

@hasheddan cool, sounds good.

not 100% sure this isn't already support but added this: https://github.com/crdsdev/doc/issues/90